indico/web/client/styles/modules/_contributions.scss

```diff
@@ -21,6 +21,12 @@
     }
 }
 
+#pub-switch {
```

Comment:
shouldn't this be a class?

Reply:
I use it for the querySelector ` const element = document.querySelector('#pub-switch');` Should I make a separate class for the styles?

Reply:
ah you query a single element; then the ID makes sense